### PR TITLE
Add visualisation route

### PR DESCRIPTION
**Pull request by cyrielo**
# Description
This change adds a link to the home page to view the heat-map of the selected project

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.
1. checkout the branch
2. Install d3-rails dependency
3. Select specific project from all projects
4. click on `Project Heatmap` link

## Impacted Areas in Application
Which pages?
- [x] Home Page
- [x] Visualisation page

# Testing Script:

### All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report
* See a link that takes you to the visualisation page

### Visualisation
* Shows a new page

# Checklist:
- [x] I have run through the testing script to make sure current functionality is unchanged
- [x] I have done relevant tests that prove my fix is effective or that my feature works
- [x] I have spent time testing out edge cases for my feature
- [x] I have updated the test script or pull request template if necessary
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/assets/javascripts/components/samples.jsx [hunk 1]
```diff
@@ -224,10 +224,24 @@ class Samples extends React.Component {
...
+                <ul>
+                  <li>
+                    <a href='/samples/new'>
+                      <i className="fa fa-flask" aria-hidden="true"></i>
+                      <span>Upload Sample</span> 
+                    </a>
+                  </li>
+                  {(this.project) ? 
+                  <li>
+                    <a href={`/projects/${this.project.id}/visuals`}>
```
Comment (yunfangjuan) [inline app/assets/javascripts/components/samples.jsx]: what does think link show when you click on it? I didn't see the code under app/views/

Comment (cyrielo) [reply app/assets/javascripts/components/samples.jsx]: Yeah it takes you to the page, I'm breaking the code into smaller units for easy review :)

Comment (yunfangjuan) [reply app/assets/javascripts/components/samples.jsx]: can you comment on the heat map link for now? you can put it back when it's ready.

**Comment by sadelokiki** [comment]
@cyrielo, Please revert the upload sample button layout to the way it is from the mock design

**Pull request merged**